code/lib/core-common/src/presets.ts

```diff
@@ -93,10 +93,10 @@ export const resolveAddonName = (
   const path = name;
 
   // when user provides full path, we don't need to do anything!
-  const managerFile = r(`${path}/manager`);
+  const managerFile = r(`${path}/manager`) || r(`${path}/manager.js`);
```

Comment:
Is it possible to try and check whether such file exists, regardless of extension, then get its full path back, with the extension? like we did for `requireResolve`. People not only do `.js`, they can do `.mjs` and maybe other extensions too

Reply:
Team decided this will be addressed later.

Reply:
We looked at this but ended up doing the opposite. We just check `/manager` now, all systems (node, vite and wp) can resolve that fine.

Reply:
Awesome!